rs/canister_sandbox/BUILD.bazel

```diff
@@ -96,7 +96,6 @@ rust_binary(
         # Keep sorted.
         ":backend_lib",
         ":build_script",
-        "@crate_index//:libc",
```

Comment:
Was this a dead dependency? Should we remove it from the relevant `Cargo.toml` as well if it's still there?

Reply:
> Was this a dead dependency?

Indeed, bazel build works without it.

> Should we remove it from the relevant Cargo.toml as well if it's still there?

We can't remove it from Cargo.toml since cargo doesn't distinguish between library and binary dependencies (at least in its current form, not sure if there's a way to achieve that analogously to bazel build files).